Plan/common/src/main/resources/assets/plan/web/js/color-selector.js

```diff
@@ -71,7 +67,11 @@
     // Change the color of the theme
     setColor(selectedColor ? selectedColor : themeDefaultColor);
 
-    let nightMode = window.localStorage.getItem('nightMode') == 'true';
+    if (selectedColor === null) {
+        window.localStorage.setItem('themeColor', currentColor);
+    }
+
+    let nightMode = window.localStorage.getItem('nightMode') ? window.localStorage.getItem('nightMode') == 'true' : '${defaultTheme}' == 'night';
```

Comment:
I think it is possible to express this `?` operator with a `||` operator instead, which would be easier to read